### DISCUSSION TITLE
How to clone and compile Advanced or Rider version?

### DISCUSSION THREAD

**Discussion by SemanticDev** [EA31337/EA31337#374] [Q&A] [answered]
Hi to All,
I cloned and compiled the repository but only the lite version was generated. How can I download the advanced version?
with respect

**Comment by kenorb** [comment, marked as answer]
After you've cloned repository recursively using Git command by:

    git clone --recursive https://github.com/EA31337/EA31337.git

You've to edit `src/include/common/mode.h` file and comment out (remove `//`) the right mode (`__advanced__` for Advanced, `__rider__` for Rider).
Then recompile the main .mq4/.mq5 EA file. In addition, reset the _Input_ parameters to default to refresh it.

**Reply by SemanticDev** [reply]
Dear kenorb,
Thank you very much for your kindness and quick response
With sincere respect
